modules/client/src/main/java/org/apache/ignite/internal/client/table/ClientTables.java

```diff
@@ -58,14 +58,14 @@ public List<Table> tables() {
     /** {@inheritDoc} */
     @Override
     public CompletableFuture<List<Table>> tablesAsync() {
-        return ch.serviceAsync(ClientOp.TABLES_GET, r -> {
+        return ch.serviceAsync(ClientOp.TABLES_GET_QUALIFIED, r -> {
```

Comment:
If we are connected to an older server (3.0.0), this will fail. We should check feature flags and fall back to the old behavior.